gems/consul/CVE-2019-16377.yml

```diff
@@ -0,0 +1,15 @@
+---
+gem: consul
+cve: 2019-16377
+url: https://github.com/makandra/consul/issues/49
+title: |
+  Consul gem insufficient authentication check: Multiple powers in one controller are not always checked correctly
+date: 2019-02-07
```

Comment:
This looks wrong. What should the date actually be?

Reply:
So sorry, I blame a lack of :coffee:.

Corrected it now.